tests/platform_tests/api/test_sfp.py

```diff
@@ -145,6 +145,18 @@ class TestSfpApi(PlatformApiTestBase):
     # Helper functions
     #
 
+    def get_sfp_attributes(self, duthost, sfp_idx, def_value, *keys):
+        if duthost.facts.get("chassis"):
+            sfps = duthost.facts.get("chassis").get("sfps")
+            if sfps:
+                sfp_value = sfps[sfp_idx]
+                for key in keys:
```

Comment:
Looping through all the interfaces would need us to add the attribute eg: "tx_disable_channel" for all interfaces. Can we add it for a particular SFP type ? 

eg: if SFP is QSFP_XX -- then tx_disable_channel.